src/scene_battle_rpg2k3.cpp

```diff
@@ -2765,6 +2766,10 @@ void Scene_Battle_Rpg2k3::CBAInit() {
 			}
 		}
 	}
+
+	if (cba_action->GetCBAMovement() != lcf::rpg::BattlerAnimationItemSkill::Movement_none) {
+		sprite->SetAfterimageAmount(cba_action->GetWeaponAnimationData()->after_image == lcf::rpg::BattlerAnimationItemSkill::AfterImage_add ? 3 : 0);
+	}
```

Comment:
When does this get reset to 0 after the movement is done?

What if you attack with after image and then..
* Escape?
* Row command?
* Change facing direction (but not movement) to attack an enemy on the other side?

Will those accidentally after-image as well?

Reply:
The afterimage is removed when the battler is idling and all afterimage-frames have the same position.

The idling state is reached directly after the CBA movement is done, so 3 frames after the afterimages are gone. It is not possible to do any actions in this 3-frame window, as they are still part of the attack handling.

Though have to check if any event command can trigger it early. Could also set the amount of afterimages to "1" when the battle action starts. This way it is always reset.

Reply:
Replaced the logic now with a fade-counter just for afterimage. This way it is ensured that the afterimage will always fade at the end of a CBA.